base_comment_template/views/comment_view.xml

```diff
@@ -2,49 +2,49 @@
 <odoo>
 
     <record model="ir.ui.view" id="view_base_comment_template_search">
-      <field name="name">base.comment.template.search</field>
-      <field name="model">base.comment.template</field>
-      <field name="arch" type="xml">
-        <search string="Comment Templates">
-          <field name="name"/>
-          <field name="position"/>
-        </search>
-      </field>
+        <field name="name">base.comment.template.search</field>
+        <field name="model">base.comment.template</field>
+        <field name="arch" type="xml">
+            <search string="Comment Templates">
+                <field name="name"/>
+                <field name="position"/>
+            </search>
+        </field>
     </record>
 
     <record model="ir.ui.view" id="view_base_comment_template_form">
-      <field name="name">base.comment.template.form</field>
-      <field name="model">base.comment.template</field>
-      <field name="arch" type="xml">
-        <form string="Comment Templates">
-          <group>
-            <field name="name"/>
-            <field name="position"/>
-            <field name="text" colspan="4"/>
-          </group>
-        </form>
-      </field>
+        <field name="name">base.comment.template.form</field>
+        <field name="model">base.comment.template</field>
+        <field name="arch" type="xml">
+            <form string="Comment Templates">
+                <group>
+                    <field name="name"/>
+                    <field name="position"/>
+                    <field name="text" colspan="4"/>
+                </group>
+            </form>
+        </field>
     </record>
 
     <record model="ir.ui.view" id="view_base_comment_template_tree">
-      <field name="name">account.comment.template.list</field>
-      <field name="model">base.comment.template</field>
-      <field name="priority" eval="6"/>
-      <field name="arch" type="xml">
-        <tree string="Comment Templates">
-          <field name="name"/>
-          <field name="position"/>
-        </tree>
-      </field>
+        <field name="name">account.comment.template.list</field>
+        <field name="model">base.comment.template</field>
+        <field name="priority" eval="6"/>
+        <field name="arch" type="xml">
+            <tree>
```

Comment:
Why are you removing the string here?

Reply:
@bjeficent if you run pylint it gives the following output:
Deprecated xml attribute "string" (xml-deprecated-tree-attribute)

Reply:
Oh I didn't know that. Thanks for the quick answer